src/pages/blog/excel-cumulative-flow-diagram.md

```diff
@@ -16,7 +17,7 @@ comments: true
 share: true
 ---
 
-[![image](/img/image_5_excel.png "image")](http://stevesmithblog.s3.amazonaws.com/SampleCumulativeFlowDiagram.zip)I’m working on a Kanban Fundamentals course that should go live soon on [Pluralsight](http://pluralsight.com). One of the topics I’m touching on is Cumulative Flow Diagrams, and how these can be used to analyze flow of a process. Some electronic kanban tools like AgileZen have some built-in support for CFD-style reports, but for instructional purposes it’s nice to be able to create one on-the-fly with whatever data one needs to illustrate a point. And of course, if you’re not using any fancy tools but would still like to benefit from the insights a cumulative flow diagram can provide, it’s nice to know how easily one can create one using Excel.
```

Comment:
That link was to a download that we should still link to from somewhere in the post (and make it clear it is a download,not just an image).

This: 
http://stevesmithblog.s3.amazonaws.com/SampleCumulativeFlowDiagram.zip

Reply:
Ok never mind we have the link in the summary.